crud/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"strings"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/app"
+	"fyne.io/fyne/v2/container"
+	"fyne.io/fyne/v2/widget"
+)
+
+var (
+	filtered = noFilter()
+	selected = -1
```

Comment:
Seems like this ought to be part of the `gui` struct instead?

Reply:
Good catch thanks. fixed